File: ClientApp/src/app/models/Flight.ts
import { AirplaneCompany } from './airplaneCompany';
import { Time } from '@angular/common';
import { Seat } from './Seat';

export class Flight
{
    id:number;
    locationFrom:string;
    locationTo:string;
    dateOfTakingOff:string;
    timeOfTakingOff:string;
    dateOfLanding:string;
    timeOfLanding:string;
    durationOfFlight:string;
    distanceOfFlight:number;
    numberOfTransfers:number;
    locationOfTransfers:string;
    priceOfTicketOfFlight:number;
    company:AirplaneCompany;
    seats:Seat[];
}

File: WebProjekat/WebProjekat/Models/Flight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebProjekat.Models
{
    public class Flight
    {
        public int Id { get; set; }
        public string LocationFrom { get; set; }
        public string LocationTo { get; set; }
        public string DateOfTakingOff { get; set; }
        public string TimeOfTakingOff { get; set; }
        public string DateOfLanding { get; set; }
        public string TimeOfLanding { get; set; }
        public string TimeOfFlight { get; set; }
        public string DurationOfFlight { get; set; }
        public string DistanceOfFlight { get; set; }
        public string NumberOfTransfers { get; set; }
        public string LocationOfTransfers { get; set; }
        public string PriceOfTicketOfFlight { get; set; }
        public virtual AirplaneCompany Company { get; set; }


    }
}
